src/js/models/v2/awards/transactions/BaseLoanTransaction.js

```diff
@@ -16,13 +15,13 @@ BaseLoanTransaction.populate = function populate(data) {
         typeDescription: data.type_description,
         actionDate: data.action_date,
         actionType: data.action_type,
-        actionTypeDescription: (data.action_type && actionTypes[data.action_type.toUpperCase()]),
+        actionTypeDescription: data.action_type_description,
```

Comment:
Let's keep this data mapping. The following award id, as an example, returns null for the description but the type is defined and the data map specifies a value.

Reply:
`http://0.0.0.0:3000/#/award/ASST_NON_DELP0000102_8900`
**Response**
![image](https://user-images.githubusercontent.com/12897813/67691753-8044e780-f975-11e9-8d19-d4613bbe11bc.png)

**Data Map**
![image](https://user-images.githubusercontent.com/12897813/67691780-876bf580-f975-11e9-836c-7a2b2b9b89cf.png)

Reply:
Interesting. Currently, the are using the base contract model 
<img width="646" alt="Screen Shot 2019-10-28 at 1 23 41 PM" src="https://user-images.githubusercontent.com/14301977/67701443-53e59700-f986-11e9-920f-ac776129318a.png">

I was trying to keep up with what is current. I will reach out to @AGuyNamedMarco to see what should be done

Reply:
It looks like both loan and contract transactions are using the core transaction maps the actionType Description to `code : description`. Reverted back to the current code. Good Catch @MaxwellKendall